fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/app/Main.java

```diff
@@ -200,6 +200,11 @@ protected Connection createConnection() {
      * can perform the DDL deployment in parallel
      */
     protected void configureConnectionPool() {
+        if ("derby".equals(dbType) && maxConnectionPoolSize > 1) {
```

Comment:
Maybe we should use the DbType.DERBY.equals to ensure consistency

Reply:
good call; in fact for enums we can just use `==`
```suggestion
        if (dbType == DbType.DERBY && maxConnectionPoolSize > 1) {
```